Tools/nuget/make_zip.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python3
+#! /usr/bin/tauthon3
```

Comment:
Yeah it's weird that they're even referring to python3 at all in the python 2.7 source, and also weird that this one uses /usr/bin/python3.